Makefile

```diff
@@ -522,6 +522,9 @@ vendor:
 		$(GO) mod vendor && \
 		$(GO) mod verify
 
+vendor-in-container:
+	podman run --privileged --rm --env HOME=/root -v `pwd`:/src -w /src docker.io/library/golang:1.12 make vendor
```

Comment:
We want to move everyone to golang 1.13.

@vrothberg Correct?

Reply:
we need first to update the CI image, if I use golang:1.13 the vendor check fails

Reply:
The CI is currently using 1.12 already which means we can't use 1.13 on our dev machines. The change here is not needed.